docs/source/apps/projinfo.rst

```diff
@@ -57,6 +58,9 @@ Synopsis
     - a PROJJSON string. The jsonschema is at https://proj.org/schemas/v0.2/projjson.schema.json (*added in 6.2*)
     - a compound CRS made from two object names separated with " + ". e.g. "WGS 84 + EGM96 height" (*added in 7.1*)
 
+    [list-crs-filter] is a comma separated combination of: auth_name=XXX,allow_deprecated,geodetic,geocentric,
```

Comment:
you can remove the explanation for list-crs-filter from here. It belongs to the one the ``list-crs`` option.

Reply:
done